perf_test/CMakeLists.txt

```diff
@@ -50,5 +50,95 @@ if (KokkosKernels_ENABLE_PERFTESTS)
     ADD_COMPONENT_SUBDIRECTORY(blas)
     ADD_SUBDIRECTORY(performance)
     #ADD_SUBDIRECTORY(common)
+
+    IF (KOKKOS_HAS_TRILINOS)
```

Comment:
Is that even defined? You should use `KOKKOSKERNELS_HAS_TRILINOS` instead

Reply:
Thanks, I changed in the commit d953f8f75e029f5929d4c02c3c3c8319a413cea3. I saw that KOKKOS_HAS_TRILINOS is used in the file kokkos-kernels/perf_test/sparse/CMakeLists.txt  too. Let me know if we should modify it too